composer.json

```diff
@@ -17,7 +17,7 @@
 		"wmde/euro": "~1.0",
 		"wmde/clock": "~1.0",
 		"wmde/fundraising-donations": "~21.0",
-		"wmde/fundraising-memberships": "~16.1",
+		"wmde/fundraising-memberships": "dev-postcode-length",
```

Comment:
this PR won't be temporary, just this branch name is temporary.
later it should be the new version (in composer.lock and maybe here as well)

Reply:
I don't really get it. Will discuss in the daily tomorrow.